src/components/HistoryPage.tsx

```
@@ -185,6 +260,43 @@ export function HistoryPage() {
+              <div className="flex items-center gap-2">
+                <FileAudio className="w-4 h-4 text-primary" />
+                <span>{audioMeta?.fileName || "history_audio.wav"}</span>
+              </div>
+              {audioMeta?.durationMs ? (
+                <span>{Math.round((audioMeta.durationMs ?? 0) / 1000)}s</span>
+              ) : null}
+            </div>
+            <audio controls autoPlay className="w-full">
+              <source src={audioUrl} type={audioMeta?.mime || "audio/wav"} />
+              Your browser does not support audio playback.
+            </audio>
```

Biome (2.1.2)

[error] Provide a track for captions when using audio or video elements.

Captions support users with hearing-impairments. They should be a transcription or translation of the dialogue, sound effects, musical cues, and other relevant audio information.

(lint/a11y/useMediaCaption)